source/cblas/cblas.di

```diff
@@ -139,333 +139,333 @@ void cblas_xerbla(blasint p, char *rout, char *form, ...);
 pure:
 
 ///
-float  cblas_sdsdot(in blasint n, in float alpha, in float *x, in blasint incx, in float *y, in blasint incy);
+float  cblas_sdsdot(blasint n, float alpha, float *x, blasint incx, float *y, blasint incy);
```

Comment:
`in` everywhere in mir means `scope const`, can't be replaced with anything else

Reply:
`in` doesn't make sense for value types, but important for pointers and ref values.

Reply:
Okay, will change